File: VueUI/src/models/Tunnel.ts
export interface Tunnel {
    id: string | null
    direction: number | null
    sshIp: string | null
    sshPort: number | null
    sshUser: string | null
    listenPort: number | null
    targetIp: string | null
    targetPort: number | null
    status: number | null
    delete: number | null
    lastAlive: string | null
}

File: poweredby-winform/Models/Tunnel.cs
using Newtonsoft.Json;

namespace poweredby_winform.Models
{
    public class Tunnel
    {

        [JsonProperty("id")]
        public string Id { get; set; }

        [JsonProperty("direction")]
        public int Direction { get; set; }

        [JsonProperty("sshIp")]
        public string SSHIp { get; set; }

        [JsonProperty("sshPort")]
        public int SSHPort { get; set; }

        [JsonProperty("sshUser")]
        public string SSHUser { get; set; }

        [JsonProperty("listenPort")]
        public int ListenPort { get; set; }

        [JsonProperty("targetIp")]
        public string TargetIp { get; set; }

        [JsonProperty("targetPort")]
        public int TargetPort { get; set; }

        [JsonProperty("status")]
        public int Status { get; set; }

        [JsonProperty("delete")]
        public int Delete { get; set; }

        [JsonProperty("lastAlive")]
        public string LastAlive { get; set; }

    }
}
